gossip/fetcher/checker.go

```diff
@@ -90,3 +92,33 @@ func (m *MockChecker) Enqueue(events inter.Events, onValidated heavycheck.OnVali
 	}
 	return nil
 }
+
```

Comment:
move MockChecker and SyncChecker into a separate file as they are for tests only.